packages/elements-dev-portal/src/version.ts

```diff
@@ -0,0 +1,2 @@
+// auto-updated during build
+export const appVersion = '1.0.0';
```

Comment:
I would just put some purposefully stupid version here, to catch if the generation breaks. Like `0.0.0` or something.

Reply:
We could but that would get updated anyway during first run anyway. And if we do it would be changed and committed within any next unrelated pr which would look weird as the version wouldn't change then. 
It would make more sense if we were unsetting that value to a placeholder after build, but I'm not sure if that's actually needed. Now we are always keeping that value up to date with actual version. Only scenario when it gets out of sync is when a release maker doesn't run build or storybook at least once, but then it would just get updated within next PR. 
Though I'm fine with adding that unsetting step in postbuild if you think it's better.

Reply:
Ah, I see. So the idea is that we *do* update this file in PRs as well. Okay. 👍